lib/http/2/compressor.rb

```diff
@@ -549,7 +551,16 @@ def header(buf)
       # @return [Array] +[[name, value], ...]+
       def decode(buf)
         list = []
-        list << @cc.process(header(buf)) until buf.empty?
+        decoded_first_regular_header = false
```

Comment:
consider changing this to:

```ruby
decoding_pseudo_headers = true
```

and switching your logic regarding this variable, it'll make it easier to read it and you can just do the assignment without the `||=`.

Reply:
@HoneyryderChuck Sounds good.